src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberAnalysisResult.cs

```diff
@@ -25,24 +29,26 @@ internal struct MemberAnalysisResult
         public readonly bool HasAnyRefOmittedArgument;
 
         private MemberAnalysisResult(MemberResolutionKind kind)
-            : this(kind, default(ImmutableArray<int>), default(ImmutableArray<int>), default(ImmutableArray<Conversion>))
+            : this(kind, constraintFailureDiagnosticsOpt: default)
```

Comment:
Not sure why `constraintFailureDiagnosticsOpt` is passed explicitly. Seems like this constructor overload can be removed.